docs/guides/tutorials/cloudflare_workers.rst

```diff
@@ -0,0 +1,258 @@
+.. _ref_guide_cloudflare_workers:
+
+==================
+Cloudflare Workers
+==================
+
+:edb-alt-title: Using EdgeDB in Cloudflare Workers
+
+
+This guide demonstrates how to integrate EdgeDB with Cloudflare Workers to 
+build serverless applications that can interact with EdgeDB. 
+
+It covers the following:
+
+- Setting up a new Cloudflare Worker project
+- Configuring EdgeDB
+- Using EdgeDB in a Cloudflare Worker
+- Deploying the Worker to Cloudflare
+
+You can use this project as a reference: `EdgeDB Cloudflare Minimal Starter`_.
+
+Prerequisites
+-------------
+
+`Sign up for a Cloudflare account`_ to later deploy your worker.
+
+Ensure you have the following installed:
+
+- `Node.js`_
+- `EdgeDB CLI`_
+
+.. _Sign up for a Cloudflare account: https://dash.cloudflare.com/sign-up
+.. _Node.js: https://nodejs.org/en/
+.. _EdgeDB CLI: https://www.edgedb.com/docs/intro/cli
+
+Setup and Configuration
+-----------------------
+
+Initialize a New Cloudflare Worker Project
+===========================================
+
+Use the `create-cloudflare`_ package to create a new Cloudflare Worker project.
+
+.. _create-cloudflare: https://www.npmjs.com/package/create-cloudflare
+
+.. code-block:: bash
+
+    $ npm create cloudflare@latest # or pnpm, yarn, bun
+
+    # or
+    $ npx create-cloudflare@latest
+
+Answer the prompts to create a new project. Pick the *"Hello World" Worker*
+template to get started.
+   
+You'll be asked if you want to put your project on Cloudflare.
+If you say yes, you'll need to sign in (if you haven't already). 
+If you don't want to deploy up right away, just go to the project folder 
+you just made to start writing your code. When you're ready to put your 
+project on Cloudflare, you can run :code:`npx wrangler deploy` to push it.
+
+.. note:: Using Wrangler CLI
+
+    If you prefer using `Wrangler`_ to set up your worker, you can use the 
+    :code:`wrangler generate` command to create a new project.
+
+.. _Wrangler: https://developers.cloudflare.com/workers/cli-wrangler
+
+
+Configure EdgeDB
+=================
+
+You can use `EdgeDB Cloud`_ for a managed service or run EdgeDB locally.
+
+.. _`EdgeDB Cloud`: https://www.edgedb.com/cloud
+
+**Local EdgeDB Setup (Optional for EdgeDB Cloud Users)**
+
+If you're running EdgeDB locally, you can use the following command 
+to create a new instance:
+
+.. code-block:: bash
+
+    $ edgedb project init
+
+It creates an :code:`edgedb.toml` config file, and a schema file 
+:code:`dbschema/default.esdl`.
+
+It also spuns up an EdgeDB instance and “links” it to the current directory. 
```

Comment:
```suggestion
It also spins up an EdgeDB instance and associates it with the current
directory.
```
Since we have a CLI command `link`, I'm thinking it might be confusing the refer to this as "link" as well.